docs/quickstart/publish.md

```diff
@@ -117,3 +118,4 @@ Possible next steps include:
 - Try fetching the pinned file by following the [retrieval quickstart](./retrieve.md).
 - If you prefer to upload a file programmatically with JavaScript or Go, check out the [Web3.storage docs](https://web3.storage/docs/how-tos/store/).
 - Alternatively, try out [Filebase](https://filebase.com/), a pinning service that offers an [AWS S3-compatible API for pinning](https://docs.filebase.com/getting-started/s3-api-getting-started-guide), so you can use any S3-compatible SDK, e.g. [aws-sdk](https://www.npmjs.com/package/aws-sdk), and [many](https://github.com/s3tools/s3cmd) more.
+- You can also try out [Functionland Fula network](https://fx.land), a pinning service that provides a true globally distributed and individually-owned nodes [Fula Network](https://docs.fx.land). The SDK supports mobile-native applications and offers built-in encryption. As an example, you can checkout the FxFotos app, which is designed as an alternative to Media Sync applications [GitHub Repo](https://github.com/functionland/fotos) and live on [Google Play](https://play.google.com/store/apps/details?id=land.fx.fotos)
```

Comment:
Duplicated content from docs/concepts/usage-ideas-examples.md, let's trim it down

```suggestion
- You can also try out [Functionland Fula network](https://fx.land), a pinning service that provides a true globally distributed and individually-owned nodes [Fula Network](https://docs.fx.land). The SDK supports [mobile-native applications](../../concepts/usage-ideas-examples.md#mobile-native-applications) and offers built-in encryption.
```